src/components/Header/Header.module.css

```diff
@@ -10,7 +10,10 @@
 .header {
   display: flex;
   align-items: center;
-  padding: var(--eds-size-1) var(--eds-size-2);
+  padding-top: var(--eds-size-1);
+  padding-right: var(--eds-size-2);
+  padding-bottom: var(--eds-size-1);
+  padding-left: var(--eds-size-2);
```

Comment:
Just curious – why was this changed?

Reply:
@dierat I changed the syntax to match the media query's. Also I was going to change this to have less padding on the left side to accommodate the menu button hover state but figured if another theme was going to pick it up later on (like Along), they might want equal heights on both sides instead of having to override that. Just felt a little weird changing it globally if say the menu button was on the right for Along for some reason.

Reply:
Awesome, thanks!